<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SizeDialog</class>
 <widget class="QDialog" name="SizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>273</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Enter start width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="wBox">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1920</number>
        </property>
        <property name="value">
         <number>640</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Enter start height:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="hBox">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1080</number>
        </property>
        <property name="value">
         <number>480</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
